app/database/db_build.sql

```diff
@@ -10,7 +10,8 @@ CREATE TABLE nuggets(
   title VARCHAR(50) NOT NULL,
   description TEXT,
   img_url VARCHAR(100),
-  author VARCHAR(30) NOT NULL
+  author VARCHAR(30) NOT NULL,
```

Comment:
we didn't want to allow such long names...
the url thing, you have a point- I don't know how long the urls generated by Amazon is but we should probably just increase it.